athena/tests/modules/modeling/module_modeling_llm/real/sample_exercises.py

```
@@ -0,0 +1,125 @@
+            if target_range in score_groups:
+                # Take the first submission from this score range
+                sampled_submissions.append(score_groups[target_range][0])
+                print(f"  {target_range}: score {score_groups[target_range][0]['score']}")
+            else:
+                print(f"  {target_range}: No submissions found")
+        
+        # Create new exercise data with sampled submissions
+        sampled_data = data.copy()
+        sampled_data['submissions'] = sampled_submissions
+        
+        return sampled_data
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)